### DISCUSSION TITLE
Use Minio instead of AWS s3 sink

### DISCUSSION THREAD

**Discussion by LinTechSo** [minio/minio#17501] [Q&A]
Hi.
if i want to send my log to MinIO instead of AWS S3 i need to add AWS s3 needed credentials like below but i dont know what can i add to them for MinIO
```
access_key_id: OK
secret_access_key: OK
assume_role: ?
credentials_file: ?
```
i think it can be fake correct ?

**Comment by harshavardhana** [comment]
Yes
